plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/server/behavior/eap/OpenShiftEapModulesController.java

```diff
@@ -50,14 +52,14 @@ private void deleteMarkers(String suffix) throws CoreException {
 		final File localDeploymentDirectory = new File(getDeploymentOptions().getDeploymentsRootFolder(true));
 		Stream.of(localDeploymentDirectory.listFiles())
 			.filter(p->p.getName().endsWith(suffix))
-			.forEach(p->p.delete());
+			.forEach(File::delete);
 	}
 	
 	private MultiStatus syncDown(IProgressMonitor monitor) throws CoreException {
 		final RSync rsync = OpenShiftServerUtils.createRSync(getServer(), monitor);
 		final File localDeploymentDirectory = new File(getDeploymentOptions().getDeploymentsRootFolder(true));
-		final MultiStatus status = new MultiStatus(OpenShiftCoreActivator.PLUGIN_ID, 0, 
-				NLS.bind("Could not sync all pods to folder {0}.", localDeploymentDirectory.getAbsolutePath()), null);
+		final MultiStatus status = new MultiStatus(OpenShiftCoreActivator.PLUGIN_ID, IStatus.OK, 
+				NLS.bind(OpenShiftCoreMessages.couldNoSyncPodsToFolder, localDeploymentDirectory.getAbsolutePath()), null);
```

Comment:
as said, I think that we should **NOT** extract string, we should strive to be consistent and remove the message classes, moving the strings back into code while using NLS.bind etc.